internal/plugin/connectors/http/generic/oauth/v1/protocol.go

```diff
@@ -0,0 +1,222 @@
+package oauth1protocol
+
+import (
+	"bytes"
+	"crypto/hmac"
+	"crypto/sha1"
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	gohttp "net/http"
+	"net/url"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type OAuth1 struct {
```

Comment:
I see a lot of uncommented public fields here. Run the code through `go lint` and/or `go fmt`